src/common/nav.js

```diff
@@ -62,22 +55,52 @@ const data = [{
     children: [{
       name: '基础表单',
       path: 'basic-form',
-      component: BasicForm,
+      component: app => dynamic({
+        app,
+        models: () => [
+          import('../models/form'),
+        ],
+        component: () => import('../routes/Forms/BasicForm'),
+      }),
     }, {
       name: '分步表单',
       path: 'step-form',
-      component: StepForm,
+      component: app => dynamic({
+        app,
+        models: () => [
+          import('../models/form'),
+        ],
+        component: () => import('../routes/Forms/StepForm'),
+      }),
       children: [{
         path: 'confirm',
-        component: Step2,
+        component: app => dynamic({
+          app,
+          models: () => [
+            import('../models/form'),
+          ],
+          component: () => import('../routes/Forms/StepForm/Step2'),
+        }),
       }, {
         path: 'result',
-        component: Step3,
+        component: app => dynamic({
```

Comment:
here return a function rather than a <Component />,so this error occurs:
Functions are not valid as a React child. This may happen if you return a Component instead of <Component /> from render. Or maybe you meant to call this function rather than return it.

Reply:
I cannot reproduce this issue? Could you show your code ?

Reply:
就是copy你的代码的pull request啊

Reply:
https://github.com/WhatAKitty/ant-design-pro please try again. This repo is ok and work good.

Reply:
And please note: If you use webpack instead of dva rodhog, please install dva@2.1.0-beta.3

Reply:
找到原因了，子组件写错了，抱歉